src/app/page/card/card.component.ts

```diff
@@ -50,7 +50,7 @@ export class CardComponent implements OnInit {
   getNextQuestion() {
     this.questionNumber++;
 
-    if (this.cards[this.questionNumber] === undefined) {
+    if (this.cards[this.questionNumber] === undefined || this.questionNumber === 10) {
```

Comment:
No magic numbers!
10 should be defined as constant and declared in some configuration file.